kafka-ui-api/src/main/resources/application.yml

```diff
@@ -1,5 +1,5 @@
 auth:
-  enabled: false
+  type: LOGIN_FORM
```

Comment:
Let's set DISABLED by default

Reply:
Done. Also added a shutdown condition if previous deprecated property is present.